shell/platform/fuchsia/flutter/engine.cc

```diff
@@ -227,8 +228,15 @@ void Engine::Initialize(
        max_frames_in_flight = product_config.get_max_frames_in_flight(),
        vsync_offset = product_config.get_vsync_offset()]() mutable {
         if (use_flatland) {
+          fidl::InterfaceHandle<fuchsia::ui::composition::Flatland> flatland;
+          zx_status_t status = fdio_service_connect(
+              "/svc/fuchsia.ui.composition.Flatland",
+              flatland.NewRequest().TakeChannel().release());
```

Comment:
Please use the "runner_services" variable here in order to connect, rather than calling fdio directly.  When using fdio directly its really hard to mock out for tests.

The preferred way would be to Connect() up above, outside of the lamba.  Create InterfaceHandle, then use the version of Connect that takes a request parameter, then pass the InterfaceHandle into this lambda and Bind() it to an InterfacePtr right here.

This makes the threading more explicit, and avoids spreading the service connection logic around so much

Reply:
Done.